config/jobs/kubernetes/sig-release/kubernetes-builds.yaml

```diff
@@ -82,7 +82,8 @@ periodics:
     preset-dind-enabled: "true"
   annotations:
     fork-per-release: "true"
-    fork-per-release-replacements: "--extra-publish-file=k8s-master -> --extra-publish-file=k8s-{{.Version}}"
+    fork-per-release-replacements: "--extra-publish-file=k8s-master -> --extra-publish-file=k8s-beta"
```

Comment:
Is there a way to validate that the format of this annotation (" <string> -> <string>") is valid. May be add the validation to the new pre-checkin job?